packages/frontend/src/components/DataViz/hooks/usePillSelectStyles.tsx

```diff
@@ -0,0 +1,58 @@
+import { createStyles } from '@mantine/core';
+
+export const usePillSelectStyles = createStyles(
```

Comment:
this helps with creating a pill select
<img width="212" alt="image" src="https://github.com/user-attachments/assets/c7a02219-05b6-49cf-84b9-b7aaf74683ee">

Reply:
Cool!